bpfman/src/oci_utils/mod.rs

```diff
@@ -20,4 +21,14 @@ pub enum ImageError {
     ByteCodeImageProcessFailure(#[from] anyhow::Error),
     #[error("BytecodeImage not found: {0}")]
     ByteCodeImageNotfound(String),
+    #[error("Failed to flush database to disk: {0}")]
```

Comment:
I'm not happy with all these error types 😉 since I don't think we ever handle the invariants. Might as well just have `DatabaseError(String)` or get fancy and have both String and optionally the source.

Reply:
Fixed up made this into `    #[error("{0}: {1}")]
    DatabaseError(String, String),`  where the first string is a context and the second is the actual error since sled::Error implements `to_string()` 👍